ccd-definition/AuthorisationCaseField/system-update.json

```diff
@@ -180,5 +180,19 @@
     "CaseFieldID": "confidentialRespondents",
     "UserRole": "caseworker-publiclaw-systemupdate",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "scannedDocuments",
+    "UserRole": "caseworker-publiclaw-systemupdate",
+    "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "evidenceHandled",
+    "UserRole": "caseworker-publiclaw-systemupdate",
+    "CRUD": "CRU"
```

Comment:
@tomaszpowroznik do we need to update these to bulkscansystemupdate?

Reply:
Yes should have `caseworker-publiclaw-bulkscansystemupdate` so automatic supplementary evideence handling can happen